x-pack/plugins/ml/public/ui_actions/open_in_anomaly_explorer_action.tsx

```diff
@@ -40,21 +114,20 @@ export function createOpenInExplorerAction(
       const [, pluginsStart] = await getStartServices();
       const locator = pluginsStart.share.url.locators.get(ML_APP_LOCATOR)!;
 
-      if (isSwimLaneEmbeddable(context)) {
-        const { embeddable, data } = context;
+      if (isSwimLaneEmbeddableContext(context)) {
+        const { data, embeddable } = context;
 
-        const { jobIds, timeRange, viewBy } = embeddable.getInput();
-        const { perPage, fromPage } = embeddable.getOutput();
+        const { localTimeRange, viewBy, jobIds, perPage, fromPage } = embeddable;
```

Comment:
update from main, localTimeRange has been renamed to timeRange$

Reply:
One more thing. embeddable.getInput().timeRange combined dashboard time range and any custom time range stored with the panel. I think when reading this you will need
`const timeRange = embeddable.timeRange$.value ?? embeddable.parentApi?.timeRange$?.value`

Reply:
Fixed in https://github.com/elastic/kibana/pull/178729/commits/4211c14b0ad753e7a92d392613b58789f9860e3f